nav/dataset/dataset_pds3_galileo_ssi.py

```
@@ -35,11 +35,11 @@ def _get_label_filespec_from_index(row: dict[str, Any]) -> str:
 
         filespec = cast(str, row['FILE_SPECIFICATION_NAME'])
         # Intentionally uppercase only
-        if not filespec.endswith('.IMG'):
+        if not filespec.endswith('.LBL'):
             raise ValueError(f'Bad Primary File Spec "{filespec}" - '
-                             'expected ".IMG"')
+                             'expected ".LBL"')
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)